src/app/modules/main/profile_section/profile/view.nim

```diff
@@ -30,8 +36,20 @@ QtObject:
       profileShowcaseAssetsModel: ProfileShowcaseAssetsModel
       profileShowcaseAssetsModelVariant: QVariant
 
+      profileShowcasePreferencesCommunitiesModel: ProfileShowcasePreferencesModel
+      profileShowcasePreferencesCommunitiesModelVariant: QVariant
+      profileShowcasePreferencesAccountsModel: ProfileShowcasePreferencesModel
+      profileShowcasePreferencesAccountsModelVariant: QVariant
+      profileShowcasePreferencesCollectiblesModel: ProfileShowcasePreferencesModel
+      profileShowcasePreferencesCollectiblesModelVariant: QVariant
+      profileShowcasePreferencesAssetsModel: ProfileShowcasePreferencesModel
+      profileShowcasePreferencesAssetsModelVariant: QVariant
+      profileShowcasePreferencesSocialLinksModel: ProfileShowcasePreferencesModel
+      profileShowcasePreferencesSocialLinksModelVariant: QVariant
```

Comment:
Does it make sense to publish the social links related stuff when it's still TODO?

Reply:
Was waiting for https://github.com/status-im/status-go/pull/4775, will update PR soon with social links soon